src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/mgmt_vmss/azuredeploy.json

```diff
@@ -90,6 +90,13 @@
         "description": "Name of load balancer backend pool."
       }
     },
+    "loadBalancerNatPoolName": {
+      "type": "string",
+      "defaultValue": "[concat(parameters('name'), 'natpool')]",
+      "metadata": {
+        "description": "Name of load balancer NAT pool."
```

Comment:
>Name of load balancer NAT pool [](start = 24, length = 30)

This seems like it should accept the name or IDs of one or more NAT pools. We have done this elsewhere.